weblate/utils/validators.py

```diff
@@ -184,28 +184,28 @@ def validate_email(value):
     if EMAIL_BLACKLIST.match(user_part):
         raise ValidationError(_('Enter a valid e-mail address.'))
     if not re.match(settings.REGISTRATION_EMAIL_MATCH, value):
-        raise ValidationError(_('This e-mail address is not allowed.'))
+        raise ValidationError(_('This e-mail address is already in use.'))
```

Comment:
This is really about not allowed addresses, it can be configured by https://docs.weblate.org/en/latest/admin/config.html#registration-email-match